database/seed.js

```diff
@@ -0,0 +1,51 @@
+
+
+// function to generate random date
+function randomDate(start, end) {
+  let d = new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime())),
```

Comment:
good that you learned how to do this, but since you are already using faker....
https://rawgit.com/Marak/faker.js/master/examples/browser/index.html#date